src/plugins/intel_cpu/tests/functional/single_layer_tests/loop.cpp

```diff
@@ -371,6 +371,65 @@ class LoopForConcatLayerCPUTest : public LoopLayerCPUTest {
     }
 };
 
+class StaticLoopDynamicSubgraphCPUTest : public SubgraphBaseTest {
+    void SetUp() override {
```

Comment:
Minor: it would be really great if you add comment with simple graph picture as in other test classes above.

Moreover this file contains classes derived from `LoopLayerCPUTest` since it's cpu specific layer tests. May I ask you change base class please? It may be done in the separate PR if we need a fix asap.

Reply:
This is a single layer test, not a subgraph test, thus no need to provide a graph scheme since it's only one layer. Regarding `LoopLayerCPUTest`, again these are single layer tests and it's completely legal to inherit the base SL class for the Loop node.